apps/mark-scan/frontend/src/components/ballot.tsx

```diff
@@ -12,6 +16,16 @@ import { ReviewScreen } from '../pages/review_screen';
 import { StartScreen } from '../pages/start_screen';
 import { ContinueToReviewPage } from '../pages/continue_to_review_page';
 
+const USER_ACTIVITY_EVENT_TYPES: EventsType[] = (() => {
+  const allEvents = new Set(DEFAULT_EVENTS);
+  // The IdlePage has an autofocus button to enable gamepad interaction,
+  // ends up triggering a focus event that would reset the idle timer.
```

Comment:
I think this comment is missing a word, maybe "that" or "which":

```
// The IdlePage has an autofocus button to enable gamepad interaction,
// which ends up triggering a focus event that would reset the idle timer.
```

Reply:
Thanks!